conda/recipes/libcudf/meta.yaml

```diff
@@ -45,6 +45,7 @@ requirements:
     - cuda-version ={{ cuda_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
+    - zlib
```

Comment:
I believe this should be moved to the `host` section.

The conda docs below describe the difference between the `host` and `build` dependency sections:

- https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#requirements-section

Even though we don't do any cross-compliation as mentioned in those docs, we've generally followed this convention for most of the RAPIDS projects.